content/admin/onboard_team/CreateUsers.md

```diff
@@ -4,41 +4,106 @@ description: "Creating Users in Contrast TeamServer"
+* Once you decide on an organization, you can choose the default **Organization Role** in the dropdown menu as well as an **Application Access Group** in the multiselect field.
+* Choose the **Date Format**, **Time Format** and **Time Zone** in the dropdown fields. 
+* The box to **Use Organization Settings** is checked by default. You can uncheck the box to create your own settings using the **Access** toggle or the box to **make user API only**. 
+* Click the **Add** button to save the user's information. 
 
->**Note:** Make sure you verify the [Role](admin-manageorgsroleperm.html#roles) so that users will have the privileges to carry out the actions you intend.
+<a href="assets/images/User-system-admin.png" rel="lightbox" title="User page"><img class="thumbnail" src="assets/images/User-system-admin.png"/></a>
+
+
+### Multiple users 
+
+To bulk add users, click the upload icon above the grid to import a spreadsheet with the users' information. The spreadsheet must be CSV or XLS format, and include the following field for each user: 
+
+* First name 
+* Last name 
+* Email 
+* Organization 
+* Organization role 
+
+You may also include the following information for each user in the spreadsheet: 
+
+* Email Activation <br> (if "None", default is **Required Password**)
+* System Admin <br> (default is **None**)
+* Application access group <br> (default is **Unassigned**)
+* Protect <br> (default is **Off**)
+* Date Format <br> (default is **MM/dd/YYYY**)
+* Time Format <br> (default is **hh:mm a**)
+* Time zone <br> (default is your time zone)
+* Use Organization Settings <br> (default is **On**)
+* Access <br> (default is **On**)
+* API only <br> (default is **Off**)
+
+> **Note:** Hover over the upload icon for a reminder of required file formats and information fields. You can also download a **CSV template** by clicking the link in the tooltip. 
+
+Once the spreadsheet upload is in progress, you can leave the page and continue with other tasks in Contrast. If the upload is successful, Contrast shows you a confirmation message with the number of users uploaded. If the upload failed, Contrast shows you an error message with the source of the error on the spreadsheet.
+
+## Organization Settings
+
+To create users as an Organization Administrator, go to the **User menu > Organization Settings > Users tab**. 
+
+### Individual users
+
+To add a single user, click the button to **Add User** above the grid, and complete the following steps: 
+
+* Enter the user's **First Name**, **Last Name** and **Email Address** in the provided fields. 
+* Choose the user's **Organization Role** in the dropdown menu. 
+* Select an **Application Access Group** to which to add the user in the dropdown menu, if desired. 
+* Choose **Date Format**, **Time Format** and **Time Zone** settings in the dropdown menus. 
+* Switch off the **Access** toggle to disable the user's access to your organization in the Contrast interface.
```

Comment:
This almost sounds like an instruction to switch off access rather than an option. Maybe we say something like "Use the Access toggle to determine whether or not the user will have access to the organization."